packages/branch-names/src/__tests__/branches.spec.ts

```diff
@@ -0,0 +1,146 @@
+import { getBranchNames } from "../branches";
+import { TestGitHubActionContext } from "@octopusdeploy/shared-action-utils";
+
+test("run with default main branch outputs correctly", () => {
```

Comment:
Might be good to have some tests for the sanitization of the branch names?

Reply:
Good poiunt